### DISCUSSION TITLE
Plugin priority and log handler

### DISCUSSION THREAD

**Discussion by joshi-mohit** [Kong/kong#10665] [Help]
I have a plugin with priority 11, 
The OIDC plugin runs with priority 1000, when the the request is sent to OIDC to be forwarded to OIDC provider. the log handler of my plugin is not invoked but the log handler of HttpLogHandler which runs at priority 12 is getting invoked.
Is there any specific thing required for this

**Comment by ghost** [comment]
> I have a plugin with priority 11,

Are you sure your plugin has been loaded correctly?

**Reply by joshi-mohit** [reply]
Yes the plugin is loaded correctly for all other traffic (which is not going OIDC provider) is working fine. If for a route there are multiple plugins do they all get executed or if a plugin like OIDC consumes it (and send a response) or redirects it to other path .. do they remaining plugins (low priority) in the chain also get executed ? because if I increase the priority of my plugin to 1001 (higher than OIDC), the data starts hitting my plugin log handler.

**Reply by ghost** [reply]
> because if I increase the priority of my plugin to 1001 (higher than OIDC), the data starts hitting my plugin log handler.

This is normal because request entered into your plugin first.

> if a plugin like OIDC consumes it (and send a response) or redirects it to other path .. do they remaining plugins (low priority) in the chain also get executed ?

Not sure, I think it probably won’t execute.
